mungegithub/mungers/sig-mention-handler.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"github.com/golang/glog"
+	//githubapi "github.com/google/go-github/github"
+	"github.com/spf13/cobra"
+	"k8s.io/test-infra/mungegithub/features"
+	github "k8s.io/test-infra/mungegithub/github"
+	//"reflect"
+	"strings"
+)
+
+var (
```

Comment:
I don't really think any of these are useful vs just putting the string literal where they are used.